conf/application.conf

```diff
@@ -73,7 +73,8 @@ microservice {
   }
 }
 
-features {
+feature-switch {
     earlyPrivateLaunch = false
     replaceIndexes = false
+    emaSupportingAgentsEnabled = true
```

Comment:
I think this should be `ema-supporting-agents-enabled` to match app-config-base and the other envs

Though annoyingly all other feature switches use camelCase. Might be something we need to change in the future back to camelCase for consistency